integration-tests/features/environment.py

```diff
@@ -15,6 +15,10 @@
 # The following API endpoint is used to check if the system is started
 _API_ENDPOINT = 'api/v1/'
 
+# Ports used by various services
+_FABRIC8_ANALYTICS_SERVER = 32000
+_FABRIC8_ANALYTICS_JOBS = 34000
+_FABRIC8_MINIO_S3 = 33000
```

Comment:
What about parametrizing using env variables?

_FABRIC8_ANALYTICS_SERVER = os.environ.get('FABRIC8_ANALYTICS_SERVER', 32000)
_FABRIC8_ANALYTICS_JOBS = os.environ.get('FABRIC8_ANALYTICS_JOBS', 34000)
_FABRIC8_MINIO_S3 = os.environ.get('FABRIC8_MINIO_S3', 33000)

I'm not strictly against this change, just a suggestion for possible next steps. WDYT?